File: meanMean.js
function getMean(arr, x, y) {
    if (x <= 1 || x > arr.length){
      return -1;
    }
    if (y <= 1 || y > arr.length){
      return -1;
    }
     let first = arr.slice(0,x);
    let second = arr.slice(arr.length -y, );
    let firstMean = (first.reduce((a,b)=> a + b)) / first.length;
    let secondMean = (second.reduce((a,b)=> a + b)) / second.length
    let totalMean = (firstMean + secondMean) / 2;
    return totalMean;
    
  }

File: mean_mean.py
def get_mean(arr,x,y):
    if x <= 1 or x > len(arr) or y <= 1 or y > len(arr):
        return -1
    first = arr[0:x]
    second = arr[-y:]
    firstMean = sum(first) / len(first)
    secondMean = sum(second) / len(second)
    totalMean = (firstMean + secondMean) / 2
    return totalMean